cpp/util/etcd.cc

```diff
@@ -628,6 +639,8 @@ void EtcdClient::WatchState::RequestDone(Status status,
   {
     // This is probably due to a timeout, just retry.
     if (!status.ok()) {
+      LOG(INFO) << "Watch request failed (" << status << "), will retry after "
```

Comment:
Yeah, WARNING and above flush, changed all these to INFO and unified the format a bit.